pkg/api/authz/authz.go

```diff
@@ -50,6 +50,7 @@ var (
 		"/api/cronjobs/",
 		"/api/mcp-catalogs",
 		"/api/mcp-catalogs/",
+		"GET /api/mcp-catalogs/{catalog_id}/entries/all-servers",
```

Comment:
I believe this is already handled by the `/api/mcp-catalogs/` (trailing slash) line above.